src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -96,6 +96,7 @@ public class DrivetrainSubsystem extends SubsystemBase {
         private ChassisSpeeds chassisSpeeds = new ChassisSpeeds(0.0, 0.0, 0.0);
 
         private Rotation2d rotationTarget;
+        private Translation2d translationTarget;
 
         private PIDController yPID = new PIDController(PROPORTIONAL_COEFFICENT, INTEGRAL_COEFFICENT,
                         DERIVATIVE_COEFFICENT);
```

Comment:
Lets rename all of these coeffs to be for either ROTATION or TRANSLATION. For rotation we can keep the values as you all have currently tuned them. For TRANSLATION as a starting point lets try to set our initial guess based on what another team used who we know also used PathPlanner and MK4 swerve last see #39 for a suggestion